install/cluster-operator/060-Deployment-strimzi-cluster-operator.yaml

```diff
@@ -105,90 +117,90 @@ spec:
               value: |
                 com.company=Red_Hat
                 rht.prod_name=Red_Hat_Integration
-                rht.prod_ver=2022.Q3
+                rht.prod_ver=2022.Q4
```

Comment:
not sure if this is going to be 2023 Q1

Reply:
Yes, I think it will be 2023.Q1. Also, is the product name now Red Hat Application Foundations?

Reply:
I'll try to find out what the product name should be